packages/docs/src/page-configs/ui-elements/scroll-container/examples/Gradient.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <va-scroll-container
+    style="max-height: 300px;"
+    color="danger"
+    vertical
+    gradient
```

Comment:
Gradient doesn't work in the Mozilla Firefox.

Reply:
Why?

<img width="1403" alt="image" src="https://user-images.githubusercontent.com/64714442/207296311-6f9416a5-7f9d-44f0-9df4-9d98fa612969.png">